pkgs/tools/networking/wireguard-go/default.nix

```diff
@@ -2,17 +2,15 @@
 
 buildGoPackage rec {
   pname = "wireguard-go";
-  version = "0.0.20200320";
+  version = "0.0.20210424";
 
   goPackagePath = "golang.zx2c4.com/wireguard";
 
   src = fetchzip {
     url = "https://git.zx2c4.com/wireguard-go/snapshot/wireguard-go-${version}.tar.xz";
-    sha256 = "0fy4qsss3i3pkq1rpgjds4aipbwlh1dr9hbbf7jn2a1c63kfks0r";
+    sha256 = "RUUueSsfEi1H+ckrnPKqbVlWONhCplMMftlyAmwK+ss=";
   };
 
-  patches = [ ./0001-Fix-darwin-build.patch ];
```

Comment:
After version update patch is not applicable, conflict
Code in the $src seems similar to result of the patch, but do not understand change in behavior

Is the correct way to check whether modified patch is needed or unpached version ok - is to check whether package builds on darwin?

Reply:
Should be fine. If not someone darwin needs to step up and fix it.